l10n_it_fatturapa_pec/models/company.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+    e_invoice_user_id = fields.Many2one(
+        "res.users", "E-invoice creator",
+        help="This user will be used at supplier e-invoice creation.",
```

Comment:
Le fatture fornitore sarebbero (e-)bill.
Oltre a bill, se si vuole esplicitare puà essere usato "Vendor bill" (usato da Odoo), oppure "Supplier bill".
In questo caso "supplier e-bill".

Stessa cosa negli altri punti del codice in cui compare (e nel readme).

Reply:
fatto